### ISSUE TITLE
Without `set guifont = Monospace\ 11`, the script fails

### ISSUE THREAD

**Issue by DaYangtuo247** [eggbean/resize-font.gvim#3]
I think I found a bug. This script doesn't work when I don't have the line `set guifont = Monospace\ 11` in my .vimrc file.

**Comment by eggbean** [comment]
Yes, there is a bug which I have meant to fix for ages. I've been meaning to look into it, so I should.

With the font that I use, only the key to reduce font size works, with increase and reset not working.
